tests/fixtures.py

```diff
@@ -12,19 +12,18 @@
 import pytest
 from unittest.mock import Mock, patch
 from configparser import ConfigParser
-import cs3.rpc.v1beta1.code_pb2 as cs3code
 import base64
 import json
 
-from cs3client import CS3Client
-from file import File
-from auth import Auth
-from user import User
-from statuscodehandler import StatusCodeHandler
-from share import Share
-from app import App
-from checkpoint import Checkpoint
-from config import Config
+from cs3client.cs3client import CS3Client
```

Comment:
I spent some time testing the different ways of doing this, this works since python can find the other package (without installing it, as you can see in the CI), the python import system is doing some magic here but I thought this was the cleanest solution.

If we wanted to use relative paths the tests would have to be part of the same package (which means we would have to use a cs3client/src/ and cs3client/tests/ type of structure and I didn't like that.